BPRBE/Constants/Values.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace BPRBE.Constants;
+
+public static class Values
+{
+    public const string MongoDbId = "_id";
+    public const string MongoDbValue = "value";
+}
```

Comment:
Can't you just put these in the one class that is actually using them? Or at least in the Persistence folder